spec/features/user_logs_in_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe User, type: :feature do
+  describe "As a registered user" do 
+    it "I can login" do 
+      user = create(:user)
+
+      visit root_path
+      click_on "Log In"
+
+      fill_in "Email", with: user.email
+      fill_in "Password", with: user.password
+      click_on "Login"
+
+      expect(current_path).to eq(user_path(user.id))
+      expect(page).to have_content(user.first_name)
+      expect(page).to have_content(user.last_name)
+      expect(page).to have_content(user.phone_number)
+      expect(page).to have_content("Sign Out")
+      expect(page).not_to have_content("Log In")
+      expect(page).not_to have_content("Create Account")
+
+    end
+
+    it "Sad Path" do 
```

Comment:
probably not, it was just a reminder that i need to do a sad path, so i'll throw one in and then re-pull request